modules/processing/network.py

```diff
@@ -371,6 +410,14 @@ def _add_dns(self, udpdata):
                 # TODO: add srv handling
                 query["answers"].append(ans)
 
+            if ( self.whitelist_enabled and 
+                (conn.get("src") in self.known_dns or conn.get("dst") in self.known_dns) and  
+                q_name in self.whitelist 
+            ):
```

Comment:
Could you reformat this a little bit? Perhaps cleaner to assign temporary `src` and `dst` variables outside of the if statement.